community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacade.java

```diff
@@ -196,7 +197,7 @@ public GraphDatabaseFacade()
     /**
      * Create a new Core API facade, backed by the given SPI.
      */
-    public void init( SPI spi, Config config )
+    public void init( SPI spi, ThreadToStatementContextBridge txBridge, Config config )
```

Comment:
Actually doing this will require to resolve again per procedure call. I thought the point of this whole exercise was to avoid having to resolve dependencies again and again?